web/src/util/api.js

```diff
@@ -0,0 +1,5 @@
+import axiosClient from 'axios';
+
+const axios = axiosClient.create({ withCredentials: true });
+
+export default axios;
```

Comment:
If we don't want to wrap up a single axios instance, we can also set `withCredentials` as a default option for all axios instances:

```
axios.defaults.withCredentials = true;
```

Then we'd have to find a home for *that*, but anyway...  😛

Reply:
yeah i saw that -- wasn't sure which one was better or where its home should be either!

Reply:
I'm on board with leaving it here for now and we can figure out a better home when we're making more than one call.  😄